server/models/trip.js

```diff
@@ -0,0 +1,56 @@
+const tripDefinition = (sequelize, DataTypes) => {
+  const Trip = sequelize.define('Trip', {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    tripType: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    from: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    to: {
+      type: DataTypes.STRING,
```

Comment:
Hi @YankyJayChris . this is a fake implantation of location. a location should be an entity. Thank you